### DISCUSSION TITLE
Reduce Addon Backup Size

### DISCUSSION THREAD

**Discussion by jensweimann** [music-assistant/support#2915] [Feature requests and ideas]
I noticed that my Home Assistant backup file size blew up quite a bit after installing music-assistant. 

I downloaded the backup file and see some big files inside there:
![{859C11F1-1FFB-44CA-B002-539362014674}](https://github.com/user-attachments/assets/7e1d6288-2ba4-48fb-8f46-865d7a981c76)

My suggestion is to allow excluding files from backup of the addon with this function:
![{02059089-E5EE-4FEC-B235-31A4B2934511}](https://github.com/user-attachments/assets/e82c0b20-f0d1-4072-9917-2a089918b589)
https://developers.home-assistant.io/docs/add-ons/configuration/

In here: https://github.com/music-assistant/home-assistant-addon/blob/main/music_assistant_dev/config.yaml

Which files are safe to exclude is a question for the devs of course. I would guess "cache.db" and "collage_images" would regenerate automatically but that's only a first idea.

**Comment by OzGav** [comment]
There have been recent efforts to reduce the size

**Comment by jensweimann** [comment]
Great news!

![image](https://github.com/user-attachments/assets/65aae6be-a061-4d81-b524-5bc9bbc1f56c)

Closed.
